### PR TITLE
fix(datasource): data type icon invisible #312

### LINKED ISSUE #312: 🐛 [Bug]: 数据源新增字段时，字段类型的图标没显示出来
### Environment

谷歌浏览器

### Version

17

### Version

3.11

### Link to minimal reproduction

数据源新增字段时，字段类型的图标没显示出来

### Step to reproduce

数据源新增字段时，字段类型的图标没显示出来
CASCLingXPC_17090044308717

### What is expected

数据源新增字段时，字段类型的图标显示出来

### What is actually happening

_No response_

### Any additional comments (optional)

_No response_

### LINKED ISSUE #325: 🐛 [Bug]: 设计态时拖动或删除一个组件，就会产生网路请求100多MB，在线上环境10MB带宽就卡死了
### Environment

chrome

### Version

latest

### Version

latest

### Link to minimal reproduction

拖动或删除一个组件，就会产生网路请求100多MB，在线上环境10MB带宽就卡死了，查看网络记录是monaco-editor产生的大量请求

### Step to reproduce

image

### What is expected

_No response_

### What is actually happening

_No response_

### Any additional comments (optional)

_No response_

### PR DESCRIPTION
**Pull request by zeyongTsai**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Background and solution
 icons config incorrect in datasource panel.  there are useful some svg icons and component that global regist in  design-core project, so we can use it.

### What is the current behavior?

icon is ok

Issue Number: #312

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by zeyongTsai** [comment]
fix #325 

Are there any areas I haven't considered or need further optimization? plz review this PR。

File: packages/plugins/datasource/src/DataSourceFieldType.vue [hunk 1]
```diff
@@ -15,7 +15,8 @@
     </div>
     <div class="field-content type-list">
       <a v-for="fieldType in state.fieldTypes" :key="fieldType.name" class="type-item" @click="selectType(fieldType)">
-        <component :is="fieldType.icon" class="type-icon"></component>
+        <!-- <component :is="fieldType.icon" class="type-icon"></component> -->
```
Comment (chilingling) [inline packages/plugins/datasource/src/DataSourceFieldType.vue]: unnecessary comment, please delete it.

File: packages/common/component/MetaBindVariable.vue [hunk 2]
```diff
@@ -6,6 +6,7 @@
   </slot>
 
   <tiny-dialog-box
+    v-if="dialogShouldInitialize"
```
Comment (chilingling) [inline packages/common/component/MetaBindVariable.vue:9]: Nice, lazyload is ok.

BTW: adding props `:destroy-on-close="true"` is also work.

Comment (zeyongTsai) [reply packages/common/component/MetaBindVariable.vue:9]: Got it, thank you. I think using lazyload is fine for now. If in actual scenarios the frequency of opening dialog is not high, switching to destroy-on-close might be better.

**Review by chilingling — APPROVED** [review]

**Review by wenmine — APPROVED** [review]
LGTM

**Pull request merged**